packages/mobile/src/escrow/saga.ts

```diff
@@ -268,16 +267,20 @@ function* doFetchSentPayments() {
     const sentPaymentsRaw = yield all(
       sentPaymentIDs.map((paymentID) => call(getEscrowedPayment, escrow, paymentID))
     )
-    const tempAddresstoRecipientPhoneNumber: Invitees = yield select(inviteesSelector)
+
+    const addressToE164Number = yield select(addressToE164NumberSelector)
     const sentPayments: EscrowedPayment[] = []
     for (let i = 0; i < sentPaymentsRaw.length; i++) {
-      const id = sentPaymentIDs[i].toLowerCase()
-      const recipientPhoneNumber = tempAddresstoRecipientPhoneNumber[id]
+      const address = sentPaymentIDs[i]
+      const recipientPhoneNumber = addressToE164Number[address]
+      const id = sentPaymentIDs[i].toLowerCase() // OPEN QUESTION: why are we making this lowercase? seems inconsistent
```

Comment:
We lower case it because data coming in/out of contracts + web3 seem to have inconsistent casing. 
Unrelated, why do you assign address to this just above and then another var for id?

Reply:
Should I be storing recipientPhoneNumber as lowercase in addressToE164Number?

That was my bad...I inserted those two lines above id without changing anything below it. I condensed it.

Reply:
> Should I be storing recipientPhoneNumber as lowercase in addressToE164Number?
Phone numbers don't have casing and you shouldn't be modifying addressToE164Number

Reply:
My bad, I misreferenced what I was talking about. However, I am calling `updateE164PhoneNumberAddresses` to store the mapping of the invitee phone number to their temporary wallet address.

https://github.com/celo-org/celo-monorepo/blob/tarikbellamine/invite-reminder-sms/packages/mobile/src/invite/saga.ts#L215